tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/LdmlConverter.java

```diff
@@ -496,6 +498,10 @@ private void processSupplemental() {
                 write(PluralRangesMapper.process(src), "misc");
                 break;
 
+            case LOCALE_DISTANCE:
```

Comment:
Hooking in the new mapper (use -DoutputTypes=LOCALE_DISTANCE to get just this data).